wine.py

```diff
@@ -267,11 +274,23 @@ def installFonts():
     winetricks_install('-q', 'settings', 'fontsmooth=rgb')
 
 
-def installD3DCompiler():
-    cmd = ['d3dcompiler_47']
-    if config.WINETRICKS_UNATTENDED is None:
-        cmd.insert(0, '-q')
-    winetricks_install(*cmd)
+def installICUDataFiles(app=None):
+    releases_url = "https://api.github.com/repos/FaithLife-Community/icu/releases"  # noqa: E501
+    json_data = utils.get_latest_release_data(releases_url)
+    icu_url = utils.get_latest_release_url(json_data)
+    # icu_tag_name = utils.get_latest_release_version_tag_name(json_data)
+    if icu_url is None:
+        logging.critical("Unable to set LogosLinuxInstaller release without URL.")  # noqa: E501
+        return
+    icu_filename = os.path.basename(icu_url)
+    utils.logos_reuse_download(
+        icu_url,
+        icu_filename,
+        config.MYDOWNLOADS,
+        app=app
+    )
+    drive_c = f"{config.INSTALLDIR}/data/wine64_bottle/drive_c"
+    utils.untar_file(f"{config.MYDOWNLOADS}/icu-win.tar.gz", drive_c)
```

Comment:
This does not work because of the gz content path. Following must be added (+ shutil imported)

shutil.copytree(f"{drive_c}/icu-win/windows", f"{drive_c}/windows", dirs_exist_ok = True)

Reply:
Does the most recent commit do what you recommend?

Reply:
@thw26 for me it still fails with:
`ModuleNotFoundError: No module named 'shutils'`

I guess you made a typo by importing `shutils` instead of `shutil`. I will try and fix it...

Reply:
See #120